src/AIView.cpp

```diff
@@ -1,17 +1,23 @@
 #include <SFML/Graphics.hpp>
 #include <SFML/Window.hpp>
-#include "GameLogic.h"
 #include "AIView.h"
 #include <iostream>
 
-AIView::AIView(GameLogic& logic): View(logic) {}
+//AIView::AIView(GameLogic& logic): View(logic) {}
 
-void AIView::init() {}
+void AIView::init() {
+    gameState = logic.getGameState();
```

Comment:
We should not set gameState when initializing the AIView because the game state is dynamic